.github/workflows/main.yml

```diff
@@ -59,17 +59,8 @@ jobs:
             ~/.cargo/git
             target
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.toml','**/Cargo.lock') }}
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
-      - name: Build and push
-        uses: docker/build-push-action@v3
-        with:
-          context: ci/
-          file: ci/Dockerfile.${{ matrix.emulator.name }}
-          tags: hwi/${{ matrix.emulator.name }}_emulator:latest
-          load: true
-          cache-from: type=gha
-          cache-to: type=gha
+      - name: Build simulator image
+        run: docker build -t hwi/${{ matrix.emulator.name }}_emulator:latest ./ci -f ci/Dockerfile.${{ matrix.emulator.name }}
```

Comment:
@notmandatory Was this required to make it work, or just an enhancement?

Reply:
Not sure if this is required but this is how it was done in the job that runs the doc tests and it looked like a simpler way to do it so adopted here too.